ca/keyreadwriter.go

```diff
@@ -36,32 +39,33 @@ type KeyReader interface {
 
 // KeyWriter writes a TLS key and cert to disk
 type KeyWriter interface {
-	Write([]byte, []byte, *KEKUpdate) error
+	Write([]byte, []byte, *KEKData) error
 	UpdateHeaders(PEMKeyHeaderManager) error
-	RotateKEK([]byte) error
+	RotateKEK(KEKData) error
 	Target() string
 }
 
-// KEKUpdate provides an optional update to the kek when writing.  The structure
+// KEKData provides an optional update to the kek when writing.  The structure
 // is needed so that we can tell the difference between "do not encrypt anymore"
 // and there is "no update".
-type KEKUpdate struct {
-	KEK []byte
+type KEKData struct {
+	KEK     []byte
+	Version uint64
```

Comment:
I was wondering whether we move this whole object to `types.proto`, so that `ca.proto`'s `GetUnlockKeyResponse` can just contain this object.